app/components/courses/entry_requirements_component.html.erb

```diff
@@ -13,9 +13,18 @@
     <% end %>
 
     <% if course.level == 'secondary' %>
-      <p class="govuk-body">
-        <%= secondary_advisory(course) %>
-      </p>
+      <% if course.engineers_teach_physics? %>
+        <p class="govuk-body">
+          Your degree subject should be in engineering, material science or a related subject, otherwise you’ll need to prove your subject knowledge in some other way.
```

Comment:
I had a quick look at our degree subject dropdown in Apply, and we seem to pluralise this as "Materials science" - which I think comes from the HECOS (HESA) list originally. Not sure whether that's how most people describe it though or not?

```suggestion
          Your degree subject should be in engineering, materials science or a related subject, otherwise you’ll need to prove your subject knowledge in some other way.
```

Reply:
@Kateharries is that fine by you?

Reply:
Yes, that's fine with me. Looks like 'materials science' is more widely used.